external/fv3fit/tests/emulation/test_zhao_carr.py

```diff
@@ -16,3 +22,50 @@ def test_classify():
         [tf.cast(classified, tf.int16) for classified in result.values()], axis=0
     )
     tf.debugging.assert_equal(class_sum, tf.ones(5, dtype=tf.int16))
+
+
+def _get_cloud_state(shape=(4, 2)):
+    return {
+        "cloud_water_mixing_ratio_input": tf.zeros(shape),
+        "cloud_water_mixing_ratio_after_gscond": tf.ones(shape),
+    }
+
+
+@pytest.mark.parametrize(
+    "classify_class", [MicrophysicsClasssesV1, MicrophysicsClassesV1OneHot],
+)
+def test_classify_classes_build(classify_class):
+
+    state = _get_cloud_state()
+
+    transform = classify_class()
+    transform = transform.build(state)
+    assert transform
+
+
+def test_classify_classes_v1():
+    state = _get_cloud_state()
```

Comment:
I didn't know sets were mutable, so I got a few fun errors related to that when using "CLASS_NAMES" as the requested variables.

Reply:
https://docs.python.org/3.8/library/stdtypes.html?highlight=frozenset#frozenset